ipfilter_test.go

```diff
@@ -728,37 +823,40 @@ func TestIpfilterParse(t *testing.T) {
 				{net.ParseIP("255.0.0.0"), net.ParseIP("255.255.255.255")},
 				{net.ParseIP("8.8.8.8"), net.ParseIP("8.8.8.8")},
 			},
-			DBHandler: &maxminddb.Reader{},
-		}},
-		{`ipfilter / {
+		}, &maxminddb.Reader{},
+		},
+		{`/ {
 			rule allow
 			ip 11.
-			}`, true, IPFConfig{
+			}`, true, IPPath{
 			PathScopes: []string{"/"},
-			Rule:       Allow,
+			IsBlock:    false,
+		}, nil,
 		},
-		},
-		{`ipfilter / {
+		{`/ {
 			rule allow
 			ip 192.168.1.10-
-			}`, true, IPFConfig{
+			}`, true, IPPath{
 			PathScopes: []string{"/"},
-			Rule:       Allow,
-		},
+			IsBlock:    false,
+		}, nil,
 		},
-		{`ipfilter / {
+		{`/ {
 			rule allow
 			ip 192.168.1.10- 20.20.20.20
-			}`, true, IPFConfig{
+			}`, true, IPPath{
 			PathScopes: []string{"/"},
-			Rule:       Allow,
-		},
+			IsBlock:    false,
+		}, nil,
 		},
 	}
 
 	for i, test := range tests {
-		c := setup.NewTestController(test.inputIpfilterConfig)
-		actualConfig, err := ipfilterParse(c)
+		c := caddy.NewTestController(test.inputIpfilterConfig)
```

Comment:
shouldn't this be `c := caddy.NewTestController("http", test.inputIpfilterConfig)` ?